src/components/pages/reusable-ui/DeleteButton.js

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from "react";
+import styled from "styled-components";
+import { TiDelete } from "react-icons/ti";
+import GlobalContext from "../../../context/GlobalContext";
+import { theme } from "../../../theme";
+
+export default function DeleteButton({ id }) {
+  const { handleDelete } = useContext(GlobalContext);
+
+  return (
+    <DeleteButtonStyled onClick={() => handleDelete(id)}>
```

Comment:
```suggestion
    <DeleteButtonStyled onClick={onClick}>
```
le handleDelete est trop specifique ici, tu l'as transpercé avec ton handleDelete ce qui ne le rend plus réutilisable